CMakeLists.txt

```diff
@@ -641,6 +642,28 @@ if (BUILD_TESTING)
     endif()
 endif()
 
+if (S2N_INTEG_TESTS)
```

Comment:
This seems to be outside of the BUILD_TESTING block. Is that important? I actually don't know what that variable is for.

Reply:
`BUILD_TESTING` allows you to disable the compilation of unit tests. It's turned on by default though. So probably makes sense to gate integ tests behind the same flag?

Reply:
Good idea, I'll move it.